samples/bluetooth/mesh/ble_peripheral_lbs_coex/prj.conf

```diff
@@ -61,3 +61,6 @@ CONFIG_BT_MESH_ONOFF_SRV=y
 # Enable the LBS service
 CONFIG_BT_LBS=y
 CONFIG_BT_LBS_POLL_BUTTON=y
+
+# Extra HCI command buffers for SMP service
+CONFIG_BT_BUF_CMD_TX_COUNT=4
```

Comment:
Does each service requires additional command buffer?

Reply:
The number 4 was based of an earlier [PR](https://github.com/nrfconnect/sdk-nrf/pull/19090/files). One per service should be right. Let me check if either of these samples need less.
Also, this is documented [here](https://github.com/zephyrproject-rtos/zephyr/blob/5dfd41e51a86c75d156241b7566ad4c1c1dce2d5/subsys/bluetooth/mesh/Kconfig#L128) but only for `BT_MESH_WORKQ_SYS` and not for `BT_MESH_WORKQ_MESH` which is default (and chosen for these samples)

Reply:
Only 3 was needed for these samples. I guess both of them only adds one additional service. Updated now.

Reply:
Any last comments about the BT_MESH_WORKQ_MESH doc as mentioned above?

Reply:
I think it is OK as it is now.